index.html

```diff
@@ -3,10 +3,203 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="dist/style.css">
   <title>Cozy House</title>
 </head>
 <body>
-  
+  <section class="start">
+    <div class="containerHeaderFooter">
```

Comment:
Actually it seems to me that there is the only `container` with `max-width: 1200px`. We can have a call and discuss this